x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/i18n.ts

```diff
@@ -42,10 +42,18 @@ export const URL_FIELD_LABEL = i18n.translate(
   'xpack.enterpriseSearch.appSearch.engine.searchUI.urlFieldLabel',
   { defaultMessage: 'URL field (Optional)' }
 );
+export const THUMBNAIL_FIELD_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.searchUI.thumbnailFieldLabel',
+  { defaultMessage: 'Thumbnail field (Optional)' }
+);
 export const URL_FIELD_HELP_TEXT = i18n.translate(
   'xpack.enterpriseSearch.appSearch.engine.searchUI.urlFieldHelpText',
   { defaultMessage: "Used as a result's link target, if applicable" }
 );
+export const THUMBNAIL_FIELD_HELP_TEXT = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.searchUI.thumbnailFieldHelpText',
+  { defaultMessage: 'Used to show a thumbnail image' }
```

Comment:
I wonder if we should clarify this to note we expect a URL? like 
```suggestion
  { defaultMessage: 'Provide an image URL to show a thumbnail image' }
```

... maybe that's too wordy though. eh, I'm probably overthinking this, feel free to ignore me

Reply:
I like that!